api/webhook/taskcluster.go

```diff
@@ -128,7 +130,7 @@ func handleStatusEvent(ctx context.Context, payload []byte) (bool, error) {
 		return false, err
 	}
 
-	if !shouldProcessStatus(&status) {
+	if !shouldProcessStatus(shared.IsFeatureEnabled(ctx, flagTaskclusterAllBranches), &status) {
```

Comment:
The first argument is `masterOnly`, so this says `IsEnabled(taskclusterAllBranches) == masterOnly`? Shouldn't it be the other way around?